release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -99,6 +99,31 @@ public function main() {
 }
 ```
 
+##### Introduction of the `function` Function Type Descriptor to Represent Any Function
+
+A new `function` function type descriptor has been introduced to represent all function values.
```

Comment:
```suggestion
A new function type descriptor named `function` has been introduced to represent all function values.
```

Reply:
I don't think saying name here is correct. This is like a built-in construct. Any way we can rephrase this without named? Can't we say
```suggestion
A new function type descriptor `function` has been introduced to represent all function values.
```

Reply:
Then it should be "A new `function` function type descriptor has been introduced to represent all function values." , since this is the convention we follow.

However, saying "function function" twice sounds a bit awkward. Do we have another way we can reword this?

Reply:
Can't we say "A new function type descriptor has been introduced to represent all function values"? Because the way we represent it is in the example.

Reply:
But I think we need to emphasize what the new type descriptor is in the description too. 

@praneesha,  when we say
```suggestion
A new `function` function type descriptor has been introduced to represent all function values.
```

the first `function` is emphasized right? So I guess it is OK? 

If not we can say
```suggestion
A new `function` type descriptor has been introduced to represent all function values.
```

Reply:
+1 for the second statement.

"A new `function` type descriptor has been introduced to represent all function values."